packages/editor/src/lib/editor/managers/FocusManager.ts

```diff
@@ -29,6 +30,18 @@ export class FocusManager {
 		if (autoFocus !== currentFocusState) {
 			editor.updateInstanceState({ isFocused: !!autoFocus })
 		}
+		this.updateContainerClass()
```

Comment:
same here

Reply:
see other comment. this line just makes sure the initial classList for the container is correct.